lib/screenplay/pending_screens_config/cache.ex

```diff
@@ -0,0 +1,123 @@
+defmodule Screenplay.PendingScreensConfig.Cache do
+  @moduledoc """
+  Functions to read data from a cached copy of the screens config.
+  """
+
+  alias ScreensConfig.{Config, Devops}
```

Comment:
For some reason, I thought it would at least need to exist because it is part of the same config struct live screens use. That's obviously not the case though. The only argument I can see for keeping it is if we wanted to verify pending screens show what we expect when the boxes are checked. I don't really see that being valuable though so I'll remove the references to it.